CMakeLists.txt

```diff
@@ -38,6 +66,7 @@ add_subdirectory(tests)
 # Documentation
 set(COMMON_PROJECT_DOMAIN ch.epfl.bluebrain)
 include(DoxygenRule)
+include(PackageConfig)
```

Comment:
Go for it! Although I don't understand the problem... A project which is not found is cloned, and this is what you want anyway if you put a top-level project inside a meta-project, no? If that's not the case, we have to document this in Subproject.cmake.